src/marshaller/HipieDDL.js

```diff
@@ -845,6 +846,7 @@
                 this.label = (visualization).label;
                 break;
             case "GRAPH":
+            case "GRAPHC":
```

Comment:
Not needed (GRAPH is a HIPIE keyword, user will set the _chartType("graph_GraohC" for this graph)